utils/formValidations.ts

```diff
@@ -159,4 +160,45 @@ export function validateFloat(float: string): boolean {
     }
 
     return true
-}
\ No newline at end of file
+}
+
+export function validateUserSubmittedLastName(name: string): boolean {
+    // The last name cannot be an empty space like " "
+    name = name.trim()
+
+    if (name.length > 75 || name === "") {
+        return false
+    }
+    return true
+}
+
+export function validateUserSubmittedFirstName(name: string): boolean {
+    // The first name is optional and may be an empty string.
+    if (name.length > 75) {
+        return false
+    }
+    return true
+}
+
+export function validateGoogleMapsUrlInput(url: string): boolean {
+    if (url.startsWith('https://') && isValidWebsite(url)) {
+        return true
```

Comment:
The isValidWebsite regex doesn't work in the Google Maps URLS, since those URLs are very long it always will fail.

Reply:
Hmm it works for the google map urls that are generated when you click the share button. Those are less than 50 characters typically.

Reply:
@Anissa3005 I'll create a new validation function for longer urls.

Reply:
@theyokohamalife if you want we can work on it together? Since I also had issues with testing the Google url, something I wanted to ask at standup

Reply:
@Anissa3005 Thanks for the offer! It should be fixed with my latest changes.